doc/development/development.rst

```diff
@@ -164,7 +164,7 @@ As described in the `black documentation <https://black.readthedocs.io/en/stable
 
 The following are some styling conventions that we follow in SpikeInterface:
 
-#. Avoid using abreviations in variable names (e.g., use :code:`recording` instead of :code:`rec`). It is specially important to avoid single letter variables.
+#. Avoid using abreviations in variable names (e.g., use :code:`recording` instead of :code:`rec`). It is especially important to avoid single letter variables.
```

Comment:
This was on me. I was not aware of that distinction. Thanks!

Reply:
@h-mayorquin In American English they are actually used pretty interchangeable, but the unofficial rule is specially is used in spoken and especially in written English. It's just convention though.

Reply:
Thanks. Useful to know.